app/views/admin/sites/_form.html.haml

```diff
@@ -18,6 +18,12 @@
   = f.text_field :ad_slot
   = f.text_area :head_tag
   = f.text_area :promotion_tag
+  = f.form_group label: {text: Site.human_attribute_name(:theme_color)} do
```

Comment:
I'm not sure about "hue".
I think we should describe following:

* What is "hue"
* This block can change theme color by "base hue" (This block can change xxx or not yyy).